src/footprint/azext_footprint/generated/_params.py

```diff
@@ -26,34 +26,34 @@ def load_arguments(self, _):
 
     with self.argument_context('footprint profile show') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('profile_name', options_list=['--name', '-n'], type=str, help='Name of the Footprint profile '
-                   'resource.', id_part='name')
+        c.argument('profile_name', options_list=['--name', '-n', '--profile-name'], type=str, help='Name of the '
+                   'Footprint profile resource.', id_part='name')
 
     with self.argument_context('footprint profile create') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('profile_name', options_list=['--name', '-n'], type=str, help='Name of the Footprint profile '
-                   'resource.')
+        c.argument('profile_name', options_list=['--name', '-n', '--profile-name'], type=str, help='Name of the '
```

Comment:
This is manually changed?

Reply:
No, the only name changes are the ones introduced when I checked in https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/footprintmonitoring/resource-manager/readme.az.md

And this time I didn't change anything manually, as linter and style passed as-is after generation.

Reply:
https://github.com/Azure/autorest.az/issues/509 the --profile-name is from kusto team, they camplain about breaking changes if we don't add the main resource name back.